README.md

```diff
@@ -2,30 +2,48 @@
 
 This is our working manifesto, which we keep as a public document. It has details specific to the product, engineering, and data teams, but a lot of it is just about how to work together effectively and bring out the best in each other.
 
-## The basics
+## The Basics
 
 Each member of the team is responsible for making this a great place to work, and for maximizing the power of technology to realize DoSomething's vision: the most young people doing the most good.
 
-### Everything is an experiment
+### Everything is an Experiment
 
-We're always looking for better ways to work, and end each sprint with a retrospective to find these improvements. The retro runs in the usual start-stop-continue style. We hold an all-teams retro, which is great for general culture and surfacing larger questions or issues. Individual teams also hold their own retros, which focus more tightly on that team's work and goals.
+We're always looking for better ways to work, and end each sprint with a retrospective to find these improvements.
 
 This entire document is just a pull request away from change. The same is true of our team structures and working arrangements. We’ve arrived at our current arrangement through steady evolution, and we’re willing to keep evolving. The qualities we work to protect have to do with how we communicate and respect each other, and how we hold each other accountable to great work and a great attitude. The rest can change.
 
 ### Diversity
 
-We know that our diversity—of gender identity, ethnicity, race, sexual orientation, age, education, religion, geography, you name it—makes us better communicators, because it reduces the chance that we'll thoughtlessly agree to a prevailing opinion. We seek out teammates who make us more diverse.
+We know that our diversity — of gender identity, ethnicity, race, sexual orientation, age, education, religion, geography, you name it — makes us better communicators, because it reduces the chance that we'll thoughtlessly agree to a prevailing opinion. We seek out teammates who make us more diverse.
 
 ### Trust
 
 As Google’s [Project Aristotle](https://www.nytimes.com/2016/02/28/magazine/what-google-learned-from-its-quest-to-build-the-perfect-team.html) shows, we succeed only when we trust one another. We understand that no one on the team is perfect. When you make a mistake, own up to it and learn from it. Trust is formed by growing and learning from mistakes as a team.
 
 ### Communication
```

Comment:
I'm not sure if this is the right section for it (maybe a Collaboration section?), or if it merits being included, but coming out of retro I keep thinking about how/when/who decides on pairing. It sounded valuable across the board, but I don't know if it's really specific to the person and/or projects that it can't be in an overarching doc like this

Reply:
Hmmmm, tell me more -- do you mean that Darwin would have a section that talks about pair programming and the value of it?

Reply:
Yea! I think it'd be useful for folks (especially new teammates) to know that it's (1) an option available to them (2) that they should be proactive and schedule it (3) the benefits/why they might want to do it.

Reply:
Going to keep it short and sweet and put it as the last paragraph of the Collaboration section:

> We value pair programming as a great option to accelerate knowledge-sharing and on-boarding, and to work through difficult and/or unfamiliar problem spaces. As the proverb goes, "A problem shared, is a problem halved."